src/main/java/graphql/schema/PropertyDataFetcher.java

```diff
@@ -53,6 +82,11 @@ private <O> PropertyDataFetcher(Function<O, T> function) {
         this.propertyName = null;
     }
 
+    private PropertyDataFetcher() {
+        this.function = null;
+        this.propertyName = null;
+    }
```

Comment:
we no longer need the property name but its kept allow older code to work that might use direct names or functions